examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -28,13 +25,14 @@ export interface IDebugViewProps {
  *
  * For the purposes of this test app, it is useful to be able to see both data sources side-by-side.
  */
-export const DebugView: React.FC<IDebugViewProps> = (props: IDebugViewProps) => {
+export const DebugView: React.FC<IDebugViewProps> = () => {
```

Comment:
(Not for this PR) I wonder if we should rename this to "ExternalDataServiceView" or something. "DebugView" isn't super descriptive.

Reply:
I agree on both -- we need to rename this as well as maybe do it in a different PR :)